src/repository/storage/postgres/orm/sequelize/note.ts

```diff
@@ -195,20 +218,31 @@ export default class NoteSequelizeStorage {
   /**
    * Gets note list by creator id
    *
-   * @param creatorId - note creator id
+   * @param userId - id of certain user
    * @param offset - number of skipped notes
    * @param limit - number of notes to get
-   * @returns { Promise<NoteList> } note
+   * @returns list of the notes
    */
-  public async getNoteListByCreatorId(creatorId: number, offset: number, limit: number): Promise<Note[]> {
+  public async getNoteListByUserId(userId: number, offset: number, limit: number): Promise<Note[]> {
+    if (this.visitsModel === null) {
+      throw new DomainError('NoteVisit model should be defined');
+    }
```

Comment:
seems like a crutch.

Reply:
we used this in all other cases, model should be initialized with value, and it is done in index file, but here by default model's value is null